catalog/resourcemanagement/resourcemanagement-querymonitor-ui/src/test/java/org/codice/ddf/resourcemanagement/query/TestQueryMonitor.java

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.resourcemanagement.query;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.codice.ddf.resourcemanagement.query.plugin.ActiveSearch;
+import org.codice.ddf.resourcemanagement.query.plugin.QueryMonitorPlugin;
+import org.codice.ddf.resourcemanagement.query.service.QueryMonitor;
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.source.Source;
+
+public class TestQueryMonitor {
+
+    private QueryMonitor queryMonitor;
+
+    private QueryMonitorPlugin queryMonitorPlugin;
+
+    private Map<UUID, ActiveSearch> activeSearchMap;
+
+    private static final String UUID_1 = "38400000-8cf0-11bd-b23e-10b96e4ef00d";
+
+    private static final String UUID_2 = "38400000-8cf0-11bd-b23e-10b96e4ef00e";
+
+    private static final String SOURCE = "Test Source";
+
+    private static final String CQL = "anyText ILIKE 'bob'";
+
+    private static final String USER_1 = "Juan";
+
+    private static final String USER_2 = "Holmes";
+
+    private static final String USER = "user";
+
+    private static final String SOURCE_ID = "sourceId";
+
+    private static final String QUERY = "query";
+
+    private static final String UUID_FIELD = "uuid";
+
+    @Before
+    public void setUp() {
+        queryMonitorPlugin = mock(QueryMonitorPlugin.class);
+        activeSearchMap = generateActiveSearchMap();
+
+        doReturn(activeSearchMap).when(queryMonitorPlugin)
+                .getActiveSearches();
+        doAnswer(invocationOnMock -> {
+            UUID uuid = (UUID) invocationOnMock.getArguments()[0];
+            return activeSearchMap.remove(uuid);
+        }).when(queryMonitorPlugin)
+                .removeActiveSearch(any(UUID.class));
+
+        doAnswer(invocationOnMock -> {
```

Comment:
Am I missing something, or are you mocking the same method you just mocked above?